MBBSEmu/Memory/IMemoryCore.cs

```diff
@@ -293,6 +293,23 @@ public interface IMemoryCore
         /// <param name="array"></param>
         void SetArray(string variableName, ReadOnlySpan<byte> array);
 
+        /// <summary>
+        ///     Writes the specified byte the specified number of times starting at the specified pointer
+        /// </summary>
+        /// <param name="segment"></param>
+        /// <param name="offset"></param>
+        /// <param name="count"></param>
+        /// <param name="value"></param>
+        void SetArray(ushort segment, ushort offset, ushort count, byte value);
+
+        /// <summary>
+        ///     Writes the specified byte the specified number of times starting at the specified pointer
+        /// </summary>
+        /// <param name="pointer"></param>
+        /// <param name="count"></param>
+        /// <param name="value"></param>
+        void SetArray(IntPtr16 pointer, ushort count, byte value);
```

Comment:
I think this would be more aptly named with `FillArray`, what do you think?

Reply:
I was tying to be consistent with the `Set` taxonomy (`SetByte`, `SetWord`, `SetArray`). But renaming it would keep the overloads simpler.